benchmarks/smallmatvec/manual_versions/libxsmm/results/results_h32_vertical.plot

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/gnuplot -persist
```

Comment:
Please add a README in this directory and use it to describe how to use the various gnuplot files.

Reply:
Done.